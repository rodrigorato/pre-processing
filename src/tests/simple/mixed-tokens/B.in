public class Foo {
    public String foo(String[] args) {
    	alias a1 = arg1;
    	var a1 = new String[](){#"#{args[0]}", #"#{args[1] + args[2]}"};
    	return a1.toString();
    }
}