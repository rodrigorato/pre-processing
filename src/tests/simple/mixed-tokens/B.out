public class Foo {
    public String foo(String[] args) {
    	
    	String[] arg1 = new String[](){"" + (args[0]) + "", "" + (args[1] + args[2]) + ""};
    	return arg1.toString();
    }
}